@xen-orchestra/lite/src/stories/web-core/copy-button/vts-copy-button.story.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <ComponentStory
+    v-slot="{ properties }"
+    :params="[prop('copy-element').type('string').preset('hello world').required().widget()]"
```

Comment:
I think it is a small oversight from an early integration:
```suggestion
    :params="[prop('value').type('string').preset('hello world').required().widget()]"
```